src/Marten/Schema/DocumentCleaner.cs

```diff
@@ -110,8 +110,8 @@ public void DeleteAllEventData()
         {
             using (var connection = _tenant.OpenConnection(CommandRunnerMode.Transactional))
             {
-                connection.Execute($"truncate table {_options.Events.DatabaseSchemaName}.mt_events cascade;" +
-                                   $"truncate table {_options.Events.DatabaseSchemaName}.mt_streams cascade");
+                connection.Execute($"drop table if exists {_options.Events.DatabaseSchemaName}.mt_events cascade;" +
```

Comment:
If you update the code as below, it will yield the same result and the unit tests will pass as well. Can you please take a look at the below:
```
connection.Execute("DO $$ BEGIN " +
                           "IF EXISTS(SELECT * FROM information_schema.tables " +
                           $"WHERE table_name = 'mt_events' AND table_schema = '{_options.Events.DatabaseSchemaName}') " +
                           $"THEN TRUNCATE TABLE {_options.Events.DatabaseSchemaName}.mt_events CASCADE; END IF;" +
                           "IF EXISTS(SELECT * FROM information_schema.tables " +
                           $"WHERE table_name = 'mt_streams' AND table_schema = '{_options.Events.DatabaseSchemaName}') " +
                           $"THEN TRUNCATE TABLE {_options.Events.DatabaseSchemaName}.mt_streams CASCADE; END IF; " +
                           "END; $$;");
```

Reply:
This is perfect. Thanks! Updated!